cmd/perceptor-scanner/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:centos7
+FROM centos:latest
```

Comment:
Why change the tag?

Reply:
Also, I don't think this allows for reproducibility of builds.

Reply:
I heard like Dave mentioned, if you use centos:latest, the nss-pem component is not there.. It may help for pivotal customers

Reply:
I don't think we need to do this because we were able to use nss-pem under a different license, AND I like build reproducibility :)

Reply:
ok @jasonf1234 reverted the changes.